### PR TITLE
 improve doc in `Adding Member Routes` section of routing chapter.

### PR DESCRIPTION
**Pull request by lanzhiheng**
Ref: https://guides.rubyonrails.org/routing.html#adding-member-routes

### CODE REVIEW

File: guides/source/routing.md [hunk 1]
```diff
@@ -519,7 +519,7 @@ resources :photos do
 end
 ```
 
-You can leave out the `:on` option, this will create the same member route except that the resource id value will be available in `params[:photo_id]` instead of `params[:id]`.
+You can leave out the `:on` option, this will create the same member route except that the resource id value will be available in `params[:photo_id]` instead of `params[:id]`, another difference is that the helper will be `photo_preview_url` and `photo_preview_path`.
```
Comment (gmcgibbon) [inline guides/source/routing.md]: I think ```. Route helpers will also be renamed from `preview_photo_url` and `preview_photo_path` to `photo_preview_url` and `photo_preview_path`.``` would sound better.

**Review by gmcgibbon — COMMENTED** [review]
Please use `[ci skip]` for doc PRs to skip Travis. Thanks!

File: guides/source/routing.md (hunk 1 above)
Comment (lanzhiheng) [reply guides/source/routing.md]: @gmcgibbon updated. Thanks for your advice.

**Review by gmcgibbon — APPROVED** [review]
Thanks for contributing!!

**Pull request merged**